packages/wonder-blocks-color/index.js

```diff
@@ -7,6 +7,7 @@ const white = "#ffffff";
 const Color = {
     // Product
     blue: "#1865f2",
+    purple: "#9059ff",
```

Comment:
Please update the `docs.md` in `wonder-blocks-color`?

Reply:
@kevinbarabash isn't it already updated? All docs.md does is iterate through all the colors on the object: `Object.keys(Color).map...`.

Reply:
My bad.  That's right.